pyopencl/cl/pyopencl-complex.h

```diff
@@ -77,6 +77,13 @@
     ; \
   } \
   \
+  inline TP TPROOT##_fma(TP a, TP b, TP c)           \
+  { \
+    return TPROOT##_new( \
+        fma(a.real, b.real, c.real) - a.imag*b.imag, \
```

Comment:
@isuruf Just curious, why isn't this?
```c
fma(fma(a.real, b.real, c.real), -a.imag, b.imag)
```

Reply:
A NEG and an FMA costs the same as a MUL and a SUB.
Also, this way the compiler is free to determine if
```
fma(fma(a.real, b.real, c.real), -a.imag, b.imag)
```
or
```
fma(fma(a.real, b.real, c.real), a.imag, -b.imag)
```
is better.

Reply:
Ah, cool! Thanks for the explanation!